src/notifier/__init__.py

```diff
@@ -51,6 +52,7 @@ class Event:
     priority: EventPriority
     service: EventService
     message: str
+    iteration: int = 0
```

Comment:
Strictly speaking no, not used. This was a half-assed attempt of surfacing the current iteration somewhere if the Events consumed somewhere actually want to display the iteration. But I'll drop it for now since it's not actually used.